listenbrainz/webserver/static/js/src/Bar.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from "react";
```

Comment:
let's also move this and `UserArtist` and related test files to a subdir, maybe `graphs` or `stats`?